server/cloud_limits.go

```diff
@@ -124,3 +124,7 @@ func isCloud(license *model.License) bool {
 func isCloudStarter(license *model.License) bool {
 	return license != nil && license.SkuShortName == "starter"
 }
+
+func isTrial(license *model.License) bool {
+	return license != nil && license.IsTrial
```

Comment:
Is there a way to distinguish what type of trial it is or is it always enterprise somehow?

Reply:
Not that I can tell. It's is an assumption I'm making -- that being a trial means it's treated as enterprise. That's seems to be what they're doing in the webapp, but it's not 1:1 because they're using webapp data, and here we're using server data.